src/addons/transitions/ReactCSSTransitionGroupChild.js

```diff
@@ -77,6 +79,10 @@ var ReactCSSTransitionGroupChild = React.createClass({
       CSSCore.removeClass(node, className);
       CSSCore.removeClass(node, activeClassName);
 
+      if (childRef.componentDidTransition) {
```

Comment:
I needed to change this to `if (childRef && childRef.componentDidTransition) {}` to prevent errors in certain situations.